ui/lib/core/addon/components/replication-summary-card.js

```diff
@@ -0,0 +1,52 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import layout from '../templates/components/replication-summary-card';
+
+/**
+ * @module ReplicationSummaryCard
+ * ReplicationSummaryCard components
+ *
+ * @example
+ * ```js
+ * <ReplicationSummaryCard
+    @title='States'
+    @replicationDetails=replicationDetails
+    />
+ * ```
+ * @param {string} [title=null] - The title to be displayed on the top left corner of the card.
+ * @param replicationDetails=null{DS.Model.replicationDetails} - An Ember data object off the Ember data model.  It is computed at the parent component and passed through to this component.
+ */
+
+export default Component.extend({
+  layout,
+  title: null,
+  replicationDetails: null,
+  lastDrWAL: computed('replicationDetails.dr.{lastWAL}', function() {
+    return this.replicationDetails.dr && this.replicationDetails.dr.lastWAL
```

Comment:
i think all of the conditionals in this component can be refactored to `return this.replicationDetails.performance.lastWAL|| 0`